src/ontology/mondo.Makefile

```diff
@@ -611,6 +611,7 @@ update-external-content-incl-rare:
 	$(MAKE) update-efo-subset -B
 	$(MAKE) update-clingen -B
 	$(MAKE) update-ordo-subsets -B
+	$(MAKE) update-omim-subsets -B
```

Comment:
@matentzn are you ok with also adding this line to the `update-external-content` goal? I know we use the combined goal where you added this line here, but to keep things consistent I think it would also be good to add to `update-external-content`.

Reply:
Looks good thank you!